scripts/config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    apiKey: '84XAr4Z2Z57+ieH6qwuIrc6zVxnlnEm/e3BnwOW72kvwebARGqXFvKE9sa/r/cJEFRHSqXCHZ9J5gkCXCQJzDNAHEsXevE/wPzgECF1YkAUmh9hoCxmqqQGlb9aF62mhEeZHHUEaqzAqux6TSuYntWu2iQC7KF4ml1G/7UagleZ0eCKoGZmTHw=='
-}
\ No newline at end of file
+    apiKey: "84XAr4Z2Z57+ieH6qwuIrc6zVxnlnEm/e3BnwOW72kvwebARGqXFvKE9sa/r/cJEFRHSqXCHZ9J5gkCXCQJzDNAHEsXevE/wPzgECF1YkAUmh9hoCxmqqQGlb9aF62mhEeZHHUEaqzAqux6TSuYntWu2iQC7KF4ml1G/7UagleZ0eCKoGZmTHw=="
```

Comment:
🤔

Reply:
@silverAndroid

Reply:
This API key isn't given from them to us so it doesn't matter and doesn't count towards some kind of quota against us. It's something we would have to change each time anyways.